src/controllers/SizeController.ts

```diff
@@ -63,250 +63,205 @@ export class SizeController
+                if (firstChild && firstChild.layout)
+                {
+                    if (!firstChild.layout.style.position)
                     {
                         childrenHeight += firstChild.height;
                     }
+                }
+                else if (firstChild instanceof Container && firstChild.height)
+                {
+                    childrenHeight += firstChild.height;
+                }
 
-                    this.layout.content.children.forEach((child) =>
+                this.layout.content.children.forEach((child) =>
+                {
+                    if (child === firstChild)
                     {
-                        if (child === firstChild)
-                        {
-                            // skip first element as it was already added
-                            return;
-                        }
+                        // skip first element as it was already added
+                        return;
+                    }
 
-                        if (child.layout && child.layout.style.position)
-                        {
-                            // skip absolute positioned elements
-                            return;
-                        }
+                    if (child.layout && child.layout.style.position)
+                    {
+                        // skip absolute positioned elements
+                        return;
+                    }
 
-                        if (child.layout)
+                    if (child.layout)
+                    {
+                        if (child.layout.style.display === 'block')
                         {
-                            if (child.layout.style.display === 'block')
-                            {
-                                childrenHeight += child.height;
-                            }
-                            else if (child.height > childrenHeight)
-                            {
-                                childrenHeight = child.height;
-                            }
+                            childrenHeight += child.height;
                         }
                         else if (child.height > childrenHeight)
                         {
                             childrenHeight = child.height;
                         }
-                    });
-
-                    if (isItJustAText(this.layout))
+                    }
+                    else if (child.height > childrenHeight)
                     {
-                        finalHeight = this.innerText?.height;
+                        childrenHeight = child.height;
                     }
+                });
 
-                    // height is basing on content height
-                    finalHeight = childrenHeight + paddingTop + paddingBottom;
-
-                    break;
-            }
-        }
-        else
-        {
-            finalHeight = getNumber(height, this.parentHeight);
-        }
```

Comment:
Those are just linting as the wrap condition was removed, the code here is the same, it's only the `else` block that is moved to
```
case 'static':
```